packages/block-editor/src/components/link-control/README.md

```diff
@@ -29,7 +19,7 @@ The distinction between the two components is perhaps best summarized by the fol
 
 When creating links the `LinkControl` component will handle two kinds of input from users:
 
-1. Entity searches - the user may input free-text based search queries for entities retrieved from remote data sources (in the context of WordPress these are `Pages`). For example, a user might search for a `Page` they have just created by name (eg: About) and the UI will return a matching result if found.
+1. Entity searches - the user may input free-text based search queries for entities retrieved from remote data sources (in the context of WordPress these are `Posts` or `Pages`). For example, a user might search for a `Page` they have just created by name (eg: About) and the UI will return a matching result if found.
```

Comment:
What I meant by `Posts` was any content type supported by WordPress that is of type `post`. It's not limited to `Posts` or `Pages`. Indeed we can see that because you can have it return `Tags` or `Categories`...etc.

How can we better clarify that?

Reply:
I switched it to "post-type entities" which is used elsewhere in the page.